packages/fxa-content-server/app/scripts/templates/partial/password-strength-balloon.mustache

```diff
@@ -1,6 +1,7 @@
 <aside id="password-strength-balloon" class="input-help input-help-balloon password-strength-balloon">
-  <h3 class="big-only">{{#t}}Your Firefox Account password{{/t}}</h3>
   <ul>
+    <li class="lock big-only">{{#unsafeTranslate}}Stay safe &mdash; don't reuse passwords.{{/unsafeTranslate}}</li>
+    <br class="big-only">
```

Comment:
🤔 can a `<br>` be a child node of `<li>`?

Reply:
it can so it is ;)

Reply:
nevermind, i misread your comment. *In* the `li` it f's up the position of the lock icon

Reply:
You read the comment that I meant to write. 😉   I wasn't sure if you can have a `<br>` as a child of a `<ul>`.